atd-kits/utils/kits_utils.py

```diff
@@ -0,0 +1,40 @@
+"""
+Utilities for interacting with the KITS advanced traffic management system's
+MSSQL database.
+"""
+import pymssql
+
+
+def get_conn(creds, max_tries=5):
+    if max_tries > 15:
+        raise Exception("Retry limit is 15")
```

Comment:
thanks for porting these verry old utils over. 15 seems like a lot. maybe 3 would be good?

since you went to the trouble of bringing these here, might as well patch them into the `signal_status_publisher.py` script as well?